src/jdk.incubator.jextract/share/classes/jdk/internal/jextract/impl/resources/C.java.template

```diff
@@ -0,0 +1,21 @@
+// Generated by jextract
```

Comment:
Not sure about using a template for this. If we do this, we severely limit ability for 3rd parties to write an annotation processor (since the annotation name/package is not stable).